Google.PowerShell/Container/GkeNodePoolCmdlets.cs

```diff
@@ -998,7 +998,7 @@ private void ProcessParams()
                     Uri nodePoolLink;
                     NodePoolName = NodePoolObject.Name;
                     if (Uri.TryCreate(NodePoolObject.SelfLink, UriKind.Absolute, out nodePoolLink)
-                        && nodePoolLink.Scheme == Uri.UriSchemeHttps)
+                        && nodePoolLink.Scheme == "https")
```

Comment:
Is there some way to still use a const here?

Reply:
Looks to me like no, unless we define one ourselves.